charts/templates/cluster-global-sidecar.yaml

```diff
@@ -0,0 +1,278 @@
+          {{- end }}
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 80
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /healthz/live
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 600
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 15
+          readinessProbe:
+            failureThreshold: 30
+            httpGet:
+              path: /healthz/ready
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 1
+            periodSeconds: 2
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            {{- toYaml $gs.resources | nindent 12 }}
+          securityContext:
+            runAsUser: 1000
+{{ range $_, $ns := $f.namespace }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: to-global-sidecar
+  namespace:  {{ $ns }}
+spec:
+  configPatches:
+    {{- range $f.wormholePort}}
+    - applyTo: VIRTUAL_HOST
+      match:
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+          vhost:
+            name: allow_any
+      patch:
+        operation: REMOVE
+    - applyTo: ROUTE_CONFIGURATION
+      match:
+        context: SIDECAR_OUTBOUND
+        routeConfiguration:
+          name: "{{ . }}"
+      patch:
+        operation: MERGE
+        value:
+          virtual_hosts:
+            {{- if $f.dispatches }}
+            {{- $svcPort := . }}
+            {{- range $f.dispatches }}
+            - domains: {{ toYaml .domains | nindent 12 }}
+              name: {{ .name }}
+              routes:
+                - match:
+                    prefix: /
+                  route:
+                    {{- if eq .cluster "_GLOBAL_SIDECAR" }}
+                    cluster: outbound|{{$svcPort}}||global-sidecar.{{ $.Values.namespace }}.svc.cluster.local
+                    {{- else }}
+                    cluster: {{ tpl .cluster $ }}
+              # (dict "fence" $f "dispatch" . "root" $ "Template" (dict "BasePath" "xx"))
+                    {{- end }}
+            {{- end }}
+            {{ else }}
+            - domains:
+              - '*'
+              name: to_global_sidecar
+              routes:
+                - match:
+                    prefix: /
+                  route:
+                    cluster: outbound|{{.}}||global-sidecar.{{ $.Values.namespace }}.svc.cluster.local
+            {{- end }}
+          request_headers_to_add:
+            - header:
+                key: "Slime-Orig-Dest"
+                value: "%DOWNSTREAM_LOCAL_ADDRESS%"
+              append: true
+            - header:
+                key: "Slime-Source-Ns"
+                value: {{ $ns }}
+              append: true
+    {{- end }}
+{{- end }}
+---
+{{- if $g }}
+{{- if $g.misc }}
+{{- if $g.misc.metric_source_type }}
```

Comment:
We should standardize the naming style of the fields later

Reply:
fix https://github.com/slime-io/slime/pull/131

Reply:
change to camel style in the future

Reply:
https://github.com/slime-io/slime/pull/132